src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/CreateVacancy.cshtml

```diff
@@ -1,6 +1,7 @@
 ﻿
 @model SFA.DAS.EmployerAccounts.Web.ViewModels.AccountDashboardViewModel
 
-<h3 class="das-panel__heading">Create an apprentice vacancy</h3>
-<p>Advertise your apprenticeship vacancies.  This is your opportunity to make your vacancy attract the best candidates.</p>
-<p><a href="#" class="govuk-link">Create a vacancy</a> </p>
\ No newline at end of file
+<h3 class="das-panel__heading">Recruitment</h3>
+<p>Recruit an apprentice by creating a vacancy. This vacancy will be advertised on Find an apprenticeship.</p>
+<p><a href="@Url.EmployerRecruitAction("create-vacancy/")" class="govuk-link">Create a vacancy</a></p>
+<p><a href="@Url.EmployerRecruitAction("")" class="govuk-link">Manage your vacancies</a> </p>
```

Comment:
if EmployerRecruitAction takes a default param = "" then no need to pass it in when targeting the recruitment base url